spec/intl.html

```diff
@@ -562,9 +563,15 @@ <h1>PartitionDateTimeRangePattern ( _dateTimeFormat_, _x_, _y_ )</h1>
           1. <ins>If ! SameTemporalType(_x_, _y_) is *false*, throw a *TypeError* exception.</ins>
         1. <ins>Let _x_ be ? HandleDateTimeValue(_dateTimeFormat_, _x_).</ins>
         1. <ins>Let _y_ be ? HandleDateTimeValue(_dateTimeFormat_, _y_).</ins>
+        1. <ins>If x.[[timeZone]] is not equal to y.[[timeZone]], then</ins>
+          1. <ins>Throw a *RangeError* exception.</ins>
+        1. <ins>Else if x.[[timeZone]] is not *undefined*, then</ins>
```

Comment:
This would need a similar change to the logic, either to include a call to DefaultTimeZone or checking whether the _dateTimeFormat_.[[TimeZone]] was specified explicitly, whichever way we decide to go with.